### ISSUE TITLE
我在自己的模型训练中使用Output-based distillation+feature distillation的结果和仅使用Output-based distillation是一样的，训练过程中的每个epoch的mAP和各个loss也基本一致。

### ISSUE THREAD

**Issue by htpauleta** [Syencil/mobile-yolov5-pruning-distillation#83]
和您readme中的（上个commit版本出现一个bug，导致精度没变其实是不对的，现已修正）一样

**Comment by htpauleta** [comment]
请问commit (fix some grad bugs) 中关于grad bugs的改动是哪些？谢谢

**Comment by htpauleta** [comment]
找到了原因，去除.detach()
